moderator/moderate/views.py

```diff
@@ -56,7 +56,60 @@ def archive(request):
 
 
 @login_required(login_url='/')
-def event(request, e_slug, q_id=None):
+def edit_event(request, slug=None):
+    """Creates a new event."""
+    user = request.user
+    query_args = {
+        'slug': slug,
+        'created_by': user
+    }
+    # Allow superusers to edit all events
+    if user.is_superuser:
+        del query_args['created_by']
+
+    event = get_object_or_404(Event, **query_args) if slug else None
+    event_form = EventForm(request.POST or None, instance=event)
+
+    if event_form.is_valid():
+        event = event_form.save(commit=False)
+        event.created_by = user
```

Comment:
Why do you want to override ``created_by`` ?

Reply:
This is a new field and it's not part of the event form.